### ISSUE TITLE
Declaration fails when Management IP already exists

### ISSUE THREAD

**Issue by simonkowallik** [F5Networks/f5-declarative-onboarding#254]
### Environment
 * Declarative Onboarding Version: 1.23.0
 * BIG-IP Version: 16.1.0

### Summary
On a BIG-IP VE which has existing Management IP configuration and admin+root password setup the DO declaration fails when it contains the same Management IP.

### Steps To Reproduce
1. Submit the following declaration:
Declaration with Management IP, which is configured on an otherwise "clean" device.

2. Observe the following error response:
HTTP requested timed out - no response from API.

Check logs: `/var/log/restjavad-audit.0.log /var/log/restjavad.0.log /var/log/ltm`:
```
# tail -20 /var/log/restjavad.0.log
                ... 8 more
        Caused by: java.lang.IllegalStateException: Expected BEGIN_ARRAY but was STRING at line 1 column 185 path $.errorStack
                at com.google.gson.stream.JsonReader.beginArray(JsonReader.java:350)
                at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:80)
                at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:61)
                at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:129)
                at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:220)
                ... 20 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
        at java.net.PlainSocketImpl.socketConnect(Native Method)
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
        at java.net.Socket.connect(Socket.java:589)
        at org.apache.thrift.transport.TSocket.open(TSocket.java:180)
        ... 13 more
```

```
# tail -f /var/log/restjavad-audit.0.log
[I][1212][11 Aug 2021 16:07:51 UTC][ForwarderPassThroughWorker] {"user":"local/admin","method":"POST","uri":"http://localhost:8100/mgmt/tm/sys/management-ip","status":409,"from":"127.0.0.1"}
[I][1213][11 Aug 2021 16:07:51 UTC][ForwarderPassThroughWorker] {"user":"local/admin","method":"POST","uri":"http://localhost:8100/mgmt/tm/sys/management-ip","status":409,"from":"127.0.0.1"}
[I][1214][11 Aug 2021 16:08:01 UTC][ForwarderPassThroughWorker] {"user":"local/admin","method":"POST","uri":"http://localhost:8100/mgmt/tm/sys/management-ip","status":409,"from":"127.0.0.1"}
```

```
 # tail /var/log/ltm
Aug 11 09:08:21 localhost.localdomain err mcpd[7999]: 01020037:3: The requested management IP (10.1.2.6) already exists.
Aug 11 09:08:21 localhost.localdomain err mcpd[7999]: 01020037:3: The requested management IP (10.1.2.6) already exists.
```

### Expected Behavior
1. declaration doesn't fail when configuring a management IP which is the same as on the system (idempotency)
2. if there is an issue with the declaration I'd expect a response pointing out the issue
3. the HTTP request is answered

### Actual Behavior
See "Steps to reproduce"

**Comment by Otimun** [comment]
If possible, I would like to add to this thread by mentioning that overall I have many different issues with configuring the management IP using Declarative Onboarding. Like @simonkowallik when configuring the same IP address in the /var/log/restnoded/restnode.log I get the errors associated with this problem:

```
Wed, 11 Aug 2021 18:47:48 GMT - finer: [f5-declarative-onboarding: restWorker.js] tryUntil error: 01020037:3: The requested management IP (10.239.27.0) already exists. tries left: 23
Wed, 11 Aug 2021 18:47:48 GMT - finest: [f5-declarative-onboarding: restWorker.js] tryUntil: retryOrReject: numRemaining: 23 , code: 409 , message: 01020037:3: The requested management IP (10.239.27.0) already exists.
Wed, 11 Aug 2021 18:47:58 GMT - fine: [f5-declarative-onboarding: restWorker.js] create localhost /tm/sys/management-ip {"name":"10.239.27.0/22","description":"configured-statically by DO"}
```

The configuration gets rolled back afterwards due to the 90 tries have been exhausted.

Later trying on a new device to first configure using DHCP and then statically configure the IP address, It still doesnt work, I get the following logs:

```
Mon, 16 Aug 2021 21:25:48 GMT - fine: [f5-declarative-onboarding: restWorker.js] [{"name":"172.28.240.74/20","description":"configured-by-dhcp"}]
Mon, 16 Aug 2021 21:26:10 GMT - fine: [f5-declarative-onboarding: restWorker.js] create localhost /tm/sys/management-ip {"name":"172.28.240.20/20","description":"configured-statically by DO"}
Mon, 16 Aug 2021 21:29:22 GMT - fine: [f5-declarative-onboarding: restWorker.js] [{"name":"172.28.240.74/20","description":"configured-by-dhcp"}]
```

The rest of the DO configuration works, but the IP doesnt change to a static IP address.

The goal: start a new device using terraform on a VM platform and configure it completely automatically without human interaction. So far all the steps before the DO declaration have been automated, but not being able to configure the IP static automatically stops me from continuing with the rest of the configuration.

**Comment by dstokesf5** [comment]
This has been resolved in the 1.26.0 release.
